### ISSUE TITLE
Updating input `choices` or `values` drops classes on child elements

### ISSUE THREAD

**Issue by nteetor** [nteetor/yonder#161]
This is definitely happening with radiobar inputs. Off the top of my head button groups are most likely affected, too, among other inputs.

**Comment by nteetor** [comment]
@jcheng5, I mentioned a context-oriented bootstrap clone and this issue illustrates why. With the current bootstrap classes styling a button group happens at the parent and child level. In {yonder} this might look like,

```R
buttonGroupInput("ex", c("Goodnight", "Moon")) %>%
  background("teal")
```

And under the hood the `background()` utility applies a class to each individual button coloring the button teal. But, if the user updates the button group the newly built child elements no longer have the proper teal background class. 

My instinct was to create new classes for applying bootstrap styles. These new classes would always be applied to the parent element. I think this still has potential. Unfortunately, I do not think theming completely resolves the issue. Even if a user could change a shade of teal they are still targeting the `.btn-teal` class and these classes are lost when using `update*Input()` functions.

**Comment by nteetor** [comment]
This will be resolved by version v0.3.0. New component level styles have been added, so child-level design classes no longer have to be applied.
